apps/server/src/modules/files/uc/sync-embedded-files.uc.ts

```diff
@@ -77,21 +115,33 @@ export class SyncEmbeddedFilesUc {
 		}
 	}
 
+	private async updateTaskLinks(file: SyncFileItem) {
+		const task = await this.embeddedFilesRepo.findTask(new ObjectId(file.parentId));
+
+		if (task) {
+			task.description = this.replaceLink(task.description, file);
+			await this.embeddedFilesRepo.updateTask(task);
+		}
```

Comment:
else {
// do something error logging? it is nice to know if we have a misstake in code
// rollback is not needed, not existing references are deleted in a additional rollout step
}